lib/services/platform-service.ts

```diff
@@ -178,7 +178,9 @@ export class PlatformService implements IPlatformService {
 			this.$pluginsService.ensureAllDependenciesAreInstalled().wait();
 			let tnsModulesDestinationPath = path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME, PlatformService.TNS_MODULES_FOLDER_NAME);
 			this.$broccoliBuilder.prepareNodeModules(tnsModulesDestinationPath, this.$projectData.projectDir, platform, lastModifiedTime).wait();
-			
+			shell.cp("-Rf", path.join(tnsModulesDestinationPath, constants.TNS_CORE_MODULES_NAME, "*"), tnsModulesDestinationPath);
+			this.$fs.deleteDirectory(constants.TNS_CORE_MODULES_NAME).wait();
```

Comment:
Would it be better if this is handled in the Broccoli pipeline, in order to avoid excessive file system access by moving files around?